playlists/excel.yaml

```diff
@@ -9,6 +9,7 @@
   group: Basics
   api_set:
     ExcelApi: 1.1
+  type: public
```

Comment:
Right, for any of those properties, you'll need to make sure they're not used by Script Lab.  But I don't think they are.  They are more probably a historical remnant.